$NetBSD: patch-plugins_xslfo_Makefile.am,v 1.1 2015/09/13 21:32:13 nros Exp $
GCC 4.6+ chokes on "--no-undefined" which older versions of GCC let fall 
to the linker.  Prefix with "-Wl," to fix build.
--- plugins/xslfo/Makefile.am.orig	2011-11-15 21:25:46.000000000 +0000
+++ plugins/xslfo/Makefile.am
@@ -18,7 +18,7 @@ xslfo_la_LIBADD = \
 xslfo_la_LDFLAGS = \
 	-avoid-version \
 	-module \
-	-no-undefined
+	-Wl,-no-undefined
 
 xslfo_la_SOURCES = 
 nodist_EXTRA_xslfo_la_SOURCES = dummy.cpp
